packages/electrode-redux-router-engine/lib/redux-router-engine.js

```diff
@@ -263,21 +263,23 @@ class ReduxRouterEngine {
       } else {
         ssrApi = withIds ? ReactDomServer.renderToString : ReactDomServer.renderToStaticMarkup;
       }
-
+      const extractor = util.getExtractor(Path.resolve("./dist/js/loadable-stats.json"));
```

Comment:
this is a hot code path, and calling `util.getExtractor`, which invokes fs sync APIs is not a good idea.

- Is a `new ChunkExtractor` needed every time?  Can it be done only once for production.
- note for that non-production a new one is needed every time in case of dev updates.

Reply:
solved, production will keep using single extractor, but dev will create new one every time